shared/video-sdk/develop/real-time-transcription/project-implementation/android.mdx

```diff
@@ -144,9 +145,9 @@ Take the following steps to start, stop and query a speech-to-text task in your
 
     A `builderToken` is usable for 5 minutes. After the time has expired, you must generate a new token.
 
-1. **Start a speech-to-text task**
+1. **Start a real-time transcription task**
 
-    To [start a speech-to-text task](#start-a-speech-to-text-task), you:
+    To start a real-time transcription task, you:
```

Comment:
loosing hyperlink here while previous doc have:
`To [start a speech-to-text task](#start-a-speech-to-text-task),`

Reply:
@saudsami  Is that fine?